### PR TITLE
fix(page): enter in title move cursor

### LINKED ISSUE #5960: Enter key not work in title
https://github.com/toeverything/blocksuite/assets/58546692/05c6b10e-a3e1-47f9-94b6-744ee8840d31

### PR DESCRIPTION
**Pull request by regischen**
close #5960

### CODE REVIEW

**Comment by AyushAgrawal-A2** [comment]
@regischen 
can we add `this.host.event.activate();` inside the `asyncFocusRichText` function... it will solve other sch scenarios as well as editor should always be active when focus text is called

File: tests/hotkey.spec.ts [hunk 1]
```diff
@@ -1300,6 +1300,20 @@ test('should cut in title works', async ({ page }) => {
   await assertRichTexts(page, ['hello']);
 });
 
+test('enter in title should move cursor in new paragraph block', async ({
+  page,
+}) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyParagraphState(page);
+  await page.pause();
```
Comment (doodlewind) [inline tests/hotkey.spec.ts:1308]: Remove this line.

**Review by doodlewind — CHANGES_REQUESTED** [review]

**Review by doodlewind — APPROVED** [review]

**Comment by doodlewind** [comment]
> @regischen 
> 
> can we add `this.host.event.activate();` inside the `asyncFocusRichText` function... it will solve other such scenarios as well, as editor should always be active when focus text is called

@regischen PTAL

**Comment by regischen** [comment]
> @regischen can we add `this.host.event.activate();` inside the `asyncFocusRichText` function... it will solve other such scenarios as well, as editor should always be active when focus text is called

I'm not sure if this is best approach. \cc @Saul-Mirone

**Pull request merged**